packages/zk-certificates/scripts/verifySC.ts

```diff
@@ -0,0 +1,27 @@
+import hre from 'hardhat';
+/**
+ * Verifies a smart contract after deployment.
```

Comment:
Thanks for adding that script. Does it work better than the verification that is part of the `deploySC` function?

Reply:
I think sometimes we might need to verify already deployed SCs.